app/components/TrackCard/tests/index.test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ *
+ * Tests for TrackCard
+ *
+ */
+
+import React from 'react';
+//import { fireEvent } from '@testing-library/dom';
+import { renderWithIntl } from '@utils/testUtils';
+import TrackCard from '../index';
+
+describe('<TrackCard />', () => {
+  const item = {
```

Comment:
Please add this to a separate file and import it here. Just keep the tests clean.